src/Ticket.java

```diff
@@ -1,31 +1,30 @@
+import java.lang.reflect.Field;
 import java.time.Instant;
+import java.util.Objects;
 
-public class Ticket {
+public class Ticket extends Identifiable {
 
-    private String id;
+    @NullableWarning
     private String concertHall;
     private String eventCode;
     private long time;
     private boolean isPromo;
     private char stadiumSector;
     private double maxBackpackWeight;
-
     private double price;
     private final long creationTime;
 
-
-
-    public  Ticket() {
+    public Ticket() {
         this.creationTime = Instant.now().getEpochSecond();
+        checkNullableWarnings(); // Check nullable warnings in the constructor
```

Comment:
this validation should be begore all this...
in your case - 18 line
And use it in every constructor